### PR TITLE
🐛 Hot-fix: Ensure synchronization between projects/workbench and projects_nodes

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

This PR fixes the issue with synchronization between the *projects* table *workbench* column and the *projects_nodes* table.
The problem arises due to the following issues:
On node creation the frontend does:
1. ```POST /projects/{project_id}/nodes``` and does not wait for an answer
2. ```PUT /projects/{project_id}``` arrives soon after with the new node in the workbench --> works since 1. call is fast

On node deletion the frontend does:
1. ```DELETE /projects/{project_id}/nodes/{node_id}``` and does not wait for an answer although that call is slow when a dynamic service needs to be removed (typically from .5s to several seconds).
2. ```PUT /projects/{project_id}``` arrives soon after with the node already removed from the workbench. the call is fast an changes *projects/workbench*.
3. the call in 1. then fails when it tries to remove the node from *projects/workbench* and *projects_nodes*. --> then the sync issue arises

This is a quick fix that could be applied to production code. the long fix will come after this one.
Also it would make sense that the frontend waits for the node creation/deletion calls before sending PUT calls.

## Related issue/s

- relates to https://github.com/ITISFoundation/osparc-issues/issues/1052

### CODE REVIEW

**Review by mguidon — COMMENTED** [review]
lgtm Thanks.

**Review by mguidon — APPROVED** [review]

File: services/web/server/src/simcore_service_webserver/projects/db.py [hunk 1]
```diff
@@ -436,90 +436,96 @@ async def replace_project(
         product_name: str,
         project_uuid: str,
     ) -> dict[str, Any]:
-        """replaces a project from a user
+        """DEPRECATED!!
```
Comment (GitHK) [inline services/web/server/src/simcore_service_webserver/projects/db.py:439]: Can we have a reason for the deprecation here?

**Review by GitHK — APPROVED** [review]

File: services/web/server/src/simcore_service_webserver/projects/db.py (hunk 1 above)
Comment (sanderegg) [reply services/web/server/src/simcore_service_webserver/projects/db.py:439]: no. the next PR will bring this in.

**Review by bisgaard-itis — APPROVED** [review]
lgtm

**Pull request merged**